### PR TITLE
GAP-2500 | decrypt editor emails in server

### PR DESCRIPTION
**Pull request by john-tco**
## Description

Ticket # and link

Decrypts scheme editor email addresses following related back-end pr: https://github.com/cabinetoffice/gap-find-admin-backend/pull/235

## Type of change

Please check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [x] Unit Test

- [ ] Integration Test (if applicable)

- [ ] End to End Test (if applicable)

## Screenshots (if appropriate):

Please attach screenshots of the change if it is a UI change:

# Checklist:

- [ ] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [ ] I have performed a self-review of my code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

### CODE REVIEW

File: packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx [hunk 1]
```diff
@@ -109,6 +109,7 @@ describe('BuildApplicationForm', () => {
   it('Should render a application form view table col', () => {
     render(
       <SchemeApplications
```
Comment (ConorFayleTCO) [inline packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx:111]: lol these should really use a function that renders the component

File: packages/admin/src/services/AdvertPageService.test.ts [hunk 2]
```diff
@@ -253,21 +259,26 @@ describe('AdvertPageService', () => {
       );
     });
 
+    //this
```
Comment (ConorFayleTCO) [inline packages/admin/src/services/AdvertPageService.test.ts:262]: remove?

Comment (john-tco) [reply packages/admin/src/services/AdvertPageService.test.ts:262]: oops

File: packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx (hunk 1 above)
Comment (john-tco) [reply packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx:111]: you're not wrong

File: packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx [hunk 3]
```diff
@@ -28,223 +30,136 @@ const getApplicationFormStats = (overrides: any = {}) =>
...
 const SCHEME_VERSION = '1';
 
+const renderComponent = (overrideProps?: Partial<SchemeApplicationsProps>) => {
+  const defaultProps = {
+    applicationForm: getSchemeApplicationsProps(),
+    applicationFormStats: getApplicationFormStats(),
+    schemeVersion: SCHEME_VERSION,
+    editorOrPublisherEmail: '',
+  };
+  render(<SchemeApplications {...{ ...defaultProps, ...overrideProps }} />);
```
Comment (jgunnCO) [inline packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx:40]:
```suggestion
  const props = {
    applicationForm: getSchemeApplicationsProps(),
    applicationFormStats: getApplicationFormStats(),
    schemeVersion: SCHEME_VERSION,
    editorOrPublisherEmail: '',
    ...overrideProps,
  };
  render(<SchemeApplications {...props} />);
```

**Review by ConorFayleTCO — APPROVED** [review]

**Review by ConorFayleTCO — APPROVED** [review]

**Pull request merged**